modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/UserCard.java

```diff
@@ -0,0 +1,173 @@
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.sample.web.internal.display.context;
+
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.LabelItem;
+
+import java.util.List;
+
+/**
+ * @author Carlos Lancha
+ */
+public class UserCard
+	implements com.liferay.frontend.taglib.clay.servlet.taglib.soy.UserCard {
+
+	public List<DropdownItem> getActionDropdownItems() {
+		return _actionDropdownItems;
+	}
+
+	public String getCssClass() {
+		return _cssClass;
+	}
+
+	public String getHref() {
+		return _href;
+	}
+
+	public String getIcon() {
+		return _icon;
+	}
+
+	public String getId() {
+		return _id;
+	}
+
+	public String getImageAlt() {
+		return _imageAlt;
+	}
+
+	public String getImageSrc() {
+		return _imageSrc;
+	}
+
+	public String getInputName() {
+		return _inputName;
+	}
+
+	public String getInputValue() {
+		return _inputValue;
+	}
+
+	public List<LabelItem> getLabels() {
+		return _labels;
+	}
+
+	public String getName() {
+		return _name;
+	}
+
+	public String getSubtitle() {
+		return _subtitle;
+	}
+
+	public String getUserColorClass() {
+		return _userColorClass;
+	}
+
+	public boolean isDisabled() {
+		return _disabled;
+	}
+
+	public boolean isSelectable() {
+		return _selectable;
+	}
+
+	public boolean isSelected() {
+		return _selected;
+	}
+
+	public void setActionDropdownItems(List<DropdownItem> actionDropdownItems) {
+		_actionDropdownItems = actionDropdownItems;
+	}
+
+	public void setCssClass(String cssClass) {
+		_cssClass = cssClass;
+	}
+
+	public void setDisabled(boolean disabled) {
+		_disabled = disabled;
+	}
+
+	public void setHref(String href) {
+		_href = href;
+	}
+
+	public void setIcon(String icon) {
+		_icon = icon;
+	}
+
+	public void setId(String id) {
+		_id = id;
+	}
+
+	public void setImageAlt(String imageAlt) {
+		_imageAlt = imageAlt;
+	}
+
+	public void setImageSrc(String imageSrc) {
+		_imageSrc = imageSrc;
+	}
+
+	public void setInputName(String inputName) {
+		_inputName = inputName;
+	}
+
+	public void setInputValue(String inputValue) {
+		_inputValue = inputValue;
+	}
+
+	public void setLabels(List<LabelItem> labels) {
+		_labels = labels;
+	}
+
+	public void setName(String name) {
+		_name = name;
+	}
+
+	public void setSelectable(boolean selectable) {
+		_selectable = selectable;
+	}
+
+	public void setSelected(boolean selected) {
+		_selected = selected;
+	}
+
+	public void setSubtitle(String subtitle) {
+		_subtitle = subtitle;
+	}
+
+	public void setUserColorClass(String userColorClass) {
+		_userColorClass = userColorClass;
+	}
+
+	private List<DropdownItem> _actionDropdownItems;
+	private String _cssClass = "custom-css-class";
+	private boolean _disabled;
+	private String _href = "user-card-href";
```

Comment:
You might want to add a `#` to the `_href` property to prevent navigating from the demo (or at least to navigate to `http://localhost:8080/#user-card-href`)

Reply:
Done, thx!